data/monitoring/service.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    prometheus.cnao.io: ""
+  name: cluster-network-addons-operator-prometheus-metrics
+  namespace: {{ .Namespace }}
+spec:
+  ports:
+    - name: metrics
+      port: 8080
```

Comment:
should this be a variable, that you set via the Render function in case the port is not the default one ?

Reply:
it does not.. I'm exposing port internal 8080 to external port 8080. the internal is already defined as port `metrics` (in the next line). this 8080 is actually a free number that I chose to be the same to avoid future confusion..